adapters/repos/db/vector/hnsw/delete_tombstone_test.go

```diff
@@ -1,168 +0,0 @@
-//                           _       _
-// __      _____  __ ___   ___  __ _| |_ ___
-// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
-//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
-//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
-//
-//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
-//
-//  CONTACT: hello@weaviate.io
-//
-
-//go:build tombstone
-// +build tombstone
-
-package hnsw
-
-import (
-	"context"
-	"fmt"
-	"log"
-	"math/rand"
-	"net/http"
-	"sync"
-	"testing"
-	"time"
-
-	"github.com/google/uuid"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
-	"github.com/sirupsen/logrus"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	"github.com/weaviate/weaviate/adapters/repos/db/vector/hnsw/distancer"
-	"github.com/weaviate/weaviate/adapters/repos/db/vector/testinghelpers"
-	"github.com/weaviate/weaviate/entities/cyclemanager"
-	ent "github.com/weaviate/weaviate/entities/vectorindex/hnsw"
-	"github.com/weaviate/weaviate/usecases/monitoring"
-)
-
-func startMonitoringForTests() (*monitoring.PrometheusMetrics, func()) {
-	mux := http.NewServeMux()
-	mux.Handle("/metrics", promhttp.Handler())
-
-	server := &http.Server{
-		Addr:    fmt.Sprintf(":%d", 2112),
-		Handler: mux,
-	}
-
-	go func() {
-		if err := server.ListenAndServe(); err != nil && err != http.ErrServerClosed {
-			log.Printf("Failed to start monitoring server: %v", err)
-		}
-	}()
-
-	stopServer := func() {
-		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-		defer cancel()
-
-		if err := server.Shutdown(ctx); err != nil {
-			log.Printf("Failed to shutdown monitoring server: %v", err)
-		}
-	}
-
-	return monitoring.GetMetrics(), stopServer
-}
-
-func TestDelete_ConcurrentWithMetrics(t *testing.T) {
```

Comment:
This test is duplicate of `TestDelete_WithConcurrentEntrypointDeletionAndTombstoneCleanup` and was excluded from pipeline test running via `tombstone` build flag